### PR TITLE
Add config values for CSI images

### PR DESCRIPTION
**Pull request by mateoflorido**
## Overview
Introduce configuration options for the CSI images.
## Rationale
We need to provide configurable options for the CSI images in the Helm chart. This pull request introduces a way to specify the image, tag, and repository for the CSI images required by the `rawfile-localpv` chart.

### CODE REVIEW

**Review by louiseschmidtgen — APPROVED** [review]
Great work, LGTM!

File: deploy/charts/rawfile-csi/Chart.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: rawfile-csi
 description: RawFile Driver Container Storage Interface
 type: application
-version: 0.8.0
+version: 1.0.0
```
Comment (neoaggelos) [inline deploy/charts/rawfile-csi/Chart.yaml]: `0.9.0`

File: deploy/charts/rawfile-csi/templates/02-storageclass.yaml [hunk 2]
```diff
@@ -4,9 +4,9 @@ kind: StorageClass
 metadata:
   name: {{ .Values.storageClass.name }}
   annotations:
-    storageclass.kubernetes.io/is-default-class: {{ .Values.storageClass.isDefault }}
+    storageclass.kubernetes.io/is-default-class: {{ .Values.storageClass.isDefault | quote }}
```
Comment (neoaggelos) [inline deploy/charts/rawfile-csi/templates/02-storageclass.yaml:7]: please remove this, we don't want possible side-effects to cause issues. if this is something we need, let's add in a separate PR

File: deploy/charts/rawfile-csi/values.yaml [hunk 3]
```diff
@@ -19,6 +19,21 @@ controller:
     - csi-driver
     - --disable-metrics
 
+csiImages:
+  repository: registry.k8s.io/sig-storage
+  csiProvisioner:
+    image: csi-provisioner
```
Comment (neoaggelos) [inline deploy/charts/rawfile-csi/values.yaml]: let's not split this in 3 separate configs for each image.

rather:
```
images:
   csiProvisionerImage: registry.k8s.io/sig-storage/csi-provisioner:v5.0.1
```

etc. this makes it easier to keep a mental model of which images we end up using at the end of the day, no need to come up with a complex config.

File: deploy/charts/rawfile-csi/templates/01-node-plugin.yaml [hunk 4]
```diff
@@ -154,7 +154,7 @@ spec:
             - name: socket-dir
               mountPath: /csi
         - name: external-snapshotter
-          image: k8s.gcr.io/sig-storage/csi-snapshotter:v6.2.1
+          image: {{ .Values.csiImages.repository }}/{{ .Values.csiImages.csiSnapshotter.image }}:{{ .Values.csiImages.csiSnapshotter.tag }}
```
Comment (neoaggelos) [inline deploy/charts/rawfile-csi/templates/01-node-plugin.yaml]: adjust according to other comment

**Review by neoaggelos — COMMENTED** [review]
Looks good, but please address comments

**Review by neoaggelos — APPROVED** [review]

**Pull request merged**